integrations/amplication/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,56 @@
+createMissingRelatedEntities: True
+resources:
+  - kind: amplication_template
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"AmplicationTemplate"'
+          properties:
+            name: .name
```

Comment:
you are mapping `.name` twice. entity by default already has title already, I think you can remove it ( also from the blueprint )